src/main/resources/assets/bloodmagic/lang/de_DE.lang

```diff
@@ -1,181 +1,256 @@
-item.bloodMagicAlchemyItem.OrbisTerrae.name=Orbis Terrae
-item.bloodMagicAlchemyItem.StrengthenedCatalyst.name=Gestärkter Katalysator
-item.bloodMagicAlchemyItem.ConcentratedCatalyst.name=Konzentierter Katalysator
-item.bloodMagicAlchemyItem.FracturedBone.name=Gebrochener Knochen
-item.bloodMagicAlchemyItem.Virtus.name=Virtus
-item.bloodMagicAlchemyItem.Reductus.name=Reductus
-item.bloodMagicAlchemyItem.Potentia.name=Potentia
-item.sanguineHelmet.name=Bluthelm
-item.itemSeerSigil.name=Siegel der Sicht
-item.itemFluidSigil.name=
+#Creative Tab
+itemGroup.BloodMagic.creativeTab=Blood Magic
 
+#Items
+item.BloodMagic.activationCrystal.weak.name=Schwacher Aktivierungskristall
+item.BloodMagic.activationCrystal.awakened.name=Erweckter Aktivierungskristall
+item.BloodMagic.activationCrystal.creative.name=Kreativer Aktivierungskristall
 
-#Creative Tab
-itemGroup.tabBloodMagic=Blood Magic
-
-#Extra Strings
-bm.string.consume=Verbrauch
-bm.string.drain=Entzug
-bm.string.tier=Stufe
-bm.string.crafting.orb.shaped=Geformte Orb-Fertigung
-bm.string.crafting.orb.shapeless=Formfreie Orb-Fertigung
+item.BloodMagic.sacrificialDagger.normal.name=Opfermesser
+item.BloodMagic.sacrificialDagger.creative.name=Kreatives Opfermesser
+item.BloodMagic.pack.selfSacrifice.name=Rucksack des Blutlassers
+item.BloodMagic.pack.sacrifice.name=Wappen
+item.BloodMagic.daggerOfSacrifice.name=Dolch der Opferung
+
+item.BloodMagic.lavaCrystal.name=Lavakristall
+
+item.BloodMagic.bound.sword.name=Gebundene Klinge
+item.BloodMagic.bound.pickaxe.name=Gebundene Spitzhacke
+item.BloodMagic.bound.axe.name=Gebundene Axt
+item.BloodMagic.bound.shovel.name=Gebundene Schaufel
+
+item.BloodMagic.bucket.lifeEssence.name=Lebenseimer
+
+item.BloodMagic.scribe.water.name=Elementar-Gravurwerkzeug: Wasser
+item.BloodMagic.scribe.fire.name=Elementar-Gravurwerkzeug: Feuer
+item.BloodMagic.scribe.earth.name=Elementar-Gravurwerkzeug: Erde
+item.BloodMagic.scribe.air.name=Elementar-Gravurwerkzeug: Leuft
+item.BloodMagic.scribe.dusk.name=Elementar-Gravurwerkzeug: Morgendämmerung
+item.BloodMagic.scribe.dawn.name=Elementar-Gravurwerkzeug: Abenddämmerung
+
+item.BloodMagic.focus.weak.name=Telepositionsfokus
+item.BloodMagic.focus.enhanced.name=Verbesserter Telepositionsfokus
+item.BloodMagic.focus.reinforced.name=Verstärkter Telepositionsfokus
+item.BloodMagic.focus.demonic.name=Dämonischer Telepositionsfokus
+
+item.BloodMagic.slate.blank.name=Leere Tafel
+item.BloodMagic.slate.reinforced.name=Verstärkte Tafel
+item.BloodMagic.slate.imbued.name=Erfüllte Tafel
+item.BloodMagic.slate.demonic.name=Dämonische Tafel
+item.BloodMagic.slate.ethereal.name=Ätherische Tafel
+
+item.BloodMagic.orb.weak.name=Schwacher Blutorb
+item.BloodMagic.orb.apprentice.name=Lehrlingsblutorb
+item.BloodMagic.orb.magician.name=Blutorb des Magiers
+item.BloodMagic.orb.master.name=Blutorb des Meisters
+item.BloodMagic.orb.archmage.name=Blutorb des Erzmagiers
+item.BloodMagic.orb.transcendent.name=Überragendes Blutorb
```

Comment:
Blutorb is a male noun, so "Überragender".
Also, I'd actually consider calling it "Transzendenter Blutorb", because it sounds so much more awesome. x3